ega-charts/localega/templates/verify-deploy.yaml

```diff
@@ -26,56 +26,84 @@ spec:
     spec:
       containers:
       - name: verify
-        image: "{{ .Values.base.image }}:{{ .Values.base.imageTag }}"
-        imagePullPolicy: {{ .Values.base.imagePullPolicy | quote }}
+        image: "{{ .Values.verify.repository }}:{{ .Values.verify.imageTag }}"
+        imagePullPolicy: {{ .Values.verify.imagePullPolicy | quote }}
         command: ["gosu", "lega", "ega-verify"]
         env:
+        - name: DEFAULT_LOG
+          value: "debug"
         - name: BROKER_CONNECTION_ATTEMPTS
           value: {{ .Values.config.broker_connection_attempts | quote }}
+        - name: BROKER_ENABLE_SSL
+          value: {{ .Values.config.broker_enable_ssl | quote }}
+        - name: BROKER_HEARTBEAT
+          value: {{ .Values.config.broker_heartbeat | quote }}
         - name: BROKER_HOST
           value: {{ if .Values.config.broker_host }}{{ .Values.config.broker_host | quote }}{{ else }}{{ template "localega.fullname" . }}-mq{{ end }}
+        - name: BROKER_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "localega.name" . }}-mq-pass
+              key: password
         - name: BROKER_PORT
           value: {{.Values.mq.port | quote }}
         - name: BROKER_RETRY_DELAY
           value: {{ .Values.config.broker_retry_delay | quote }}
         - name: BROKER_USERNAME
           value: {{ .Values.config.broker_username | quote }}
-        - name: KEYSERVER_ENDPOINT
-          value: {{ if .Values.config.keyserver_host }}{{ .Values.config.keyserver_host }}{{ else }}https://{{ template "localega.fullname" . }}-keys{{ end }}:{{ .Values.keys.port}}{{ .Values.config.keyserver_endpoint}}
-        - name: KEYSERVER_HOST
-          value: {{ if .Values.config.keyserver_host }}{{ .Values.config.keyserver_host | quote }}{{ else }}{{ template "localega.fullname" . }}-keys{{ end }}
-        - name: KEYSERVER_PORT
-          value: {{ .Values.keys.port | quote }}
+        - name: BROKER_VHOST
+          value: "/"
         - name: POSTGRES_HOST
           value: {{ if .Values.config.postgres_host }}{{ .Values.config.postgres_host | quote }}{{ else }}{{ template "localega.fullname" . }}-db{{ end }}
         - name: POSTGRES_PASSWORD
           valueFrom:
             secretKeyRef:
               name: {{ template "localega.name" . }}-db-secret
               key: postgres_password
+        - name: POSTGRES_SSLMODE
+          value: "disable"
```

Comment:
`{{ .Values.config.broker_enable_ssl | quote }}`

Reply:
`{{ .Values.config.postgres_sslmode | quote }}` - mistake meant this :disappointed: